wiki/Guides/Compressing_files/fr.md

```diff
@@ -0,0 +1,99 @@
+
+Il existe 2 types de compression. La compression **avec perte** et la compression **sans perte**.
+
+- **Sans perte** implique que la qualité ne se dégrade jamais et peut être compressée ou décompressée à plusieurs reprises.
+- **Avec perte** est une forme de compression beaucoup plus puissante qui sacrifie la qualité à la demande d'espace/de processeur.
+
+## Vidéo
+
+Les vidéos sont généralement l'endroit où les gens se trompent. Il existe des milliers de codecs vidéo, et *croyez-le ou non, YouTube est généralement un endroit terrible pour obtenir des vidéos*.
+
+Le H.264 est un bon codec à utiliser, mais comme la plupart des codecs vidéo, il est avec perte. Cela signifie que vous devez éviter de ré-encoder la vidéo plusieurs fois, mais plutôt encoder une fois à partir de la source haute résolution.
+
+### Utilisation de Handbrake
+
+Cette section vous montre comment supprimer le son des vidéos en utilisant [Handbrake](https://handbrake.fr/) pour réduire la taille du fichier vidéo.
+
+1. Ouvrez Handbrake et importez votre fichier vidéo. Vous pouvez glisser et déposer votre fichier dans Handbrake ou l'importer manuellement en cliquant sur l'option `Fichier`.
+
+![Importation de fichiers vidéo dans Handbrake](img/import-handbrake-FR.png "Importation de la vidéo dans Handbrake")
+
+2. Sélectionnez le préréglage `Fast 720p30`.
+
+![Sélection du préréglage vidéo](img/preset-handbrake-FR.png "Sélection du préréglage")
+
+3. Sélectionnez l'onglet `Audio` et supprimez toutes les pistes audio. Faites de même pour les sous-titres en allant dans l'onglet `Sous-titres` et en supprimant toutes les entrées.
+
+![Suppression des pistes audio de l'onglet audio dans Handbrake](img/removeaudio-handbrake-FR.png "Suppression des pistes audio")
+
+4. Allez dans l'onglet `Vidéo` et définissez le codec vidéo comme `H.264 (X264)`. Modifiez la valeur de `Qualité constante` entre 20 et 25. Des valeurs plus petites produiront des fichiers de plus petite taille au détriment de la qualité de la vidéo.
+5. Si vous êtes prêt à passer plus de temps à encoder, changez le `Préréglage de l'encodeur` sous `Options de l'encodeur`. Les préréglages plus lents offrent une meilleure qualité vidéo et peuvent également réduire la taille du fichier vidéo, mais ne descendez pas jusqu'au placebo car cela prend beaucoup plus de temps que `VerySlow` pour une très faible amélioration de la qualité. Réglez le framerate pour qu'il soit le même que celui de la source et réglez-le sur un framerate constant.
+
+![Réglage du codec vidéo et de la qualité dans Handbrake](img/codecquality-handbrake-FR.png "Réglage du codec vidéo et de la qualité constante")
+
+6. Pour redimensionner l'image du fichier vidéo, allez dans l'onglet `Dimensions` et changez la largeur en `1280` et la hauteur en `720`.
+
+![Réglage des dimensions de la vidéo et de la qualité dans Handbrake](img/dimensions-handbrake-FR.png "Définition des dimensions de la vidéo")
```

Comment:
i understand why it's there, but the bottom red rectangle is not necessary, because the top highlighted option is the only one you'd need to change (most of the time)

![image](https://user-images.githubusercontent.com/36758269/145277128-f8606c96-fa90-4a1d-8d8e-31603039bb9e.png)

Reply:
it might be okay to highlight the bottom part since it references the final video size, in which case *maybe*, but do make the same edit on the english version if you decide to keep this